Sources/SocksCore/FDSet.swift

```diff
@@ -31,7 +31,11 @@ import Darwin
 
 func fdZero(_ set: inout fd_set) {
 #if os(Linux)
+  set.__fds_bits = (0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0)
```

Comment:
I just noticed that when the CI complained :)  I think I've fixed it up.  I re-built on arm successfully, but I'd feel better if the CI gave its blessing.